src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -53,6 +54,11 @@ export const RoomModalContent: React.FC<RoomModalContentProps> = ({
 
   const { enterRoom, recentRoomUsers } = useRoom({ room, venueName });
 
+  const [enterRoomWithSound] = useCustomSound(room.enterSound, {
+    interrupt: true,
+    onend: enterRoom,
+  });
```

Comment:
[TODO] Fix bug where we can't enter the room anymore if there is no `enterSound` defined for this room.

---

[note] We use `onend` here so that we wait until the sound has finished playing before entering the room.

Reply:
`onend => onEnd`?

Reply:
Nope. For whatever reason that’s what howler chose to use as their API here.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1269/commits/ce530ff4e0e30ad24826b7e12174790e71307670